docs/src/literate-tutorials/ns_vs_diffeq.jl

```diff
@@ -1,6 +1,14 @@
+# Putting this flag to false reproduces the figure shown in the example #src
+# We check for laminar flow development in the CI                       #src
```

Comment:
We should either update this comment or really check for the velocity profile in the test.

Reply:
```suggestion
# We check for a divergence free velocity field in the CI                      #src
```
?

Reply:
That is fine for me.